loader_hub/notion/base.py

```diff
@@ -90,21 +90,32 @@ def read_page(self, page_id: str) -> str:
         """Read a page."""
         return self._read_block(page_id)
 
-    def query_database(
-        self, database_id: str, query_dict: Dict[str, Any] = {}
-    ) -> List[str]:
+    def query_database(self, database_id: str) -> List[str]:
         """Get all the pages from a Notion database."""
+        query_dict = {"page_size": 100}
```

Comment:
I think this approach is fine but it's best to keep query_dict as an argument you can pass in still. Perhaps it can just default to {"page_size": 100}

Reply:
I agree :)